src/app/courses/course/course.component.scss

```diff
@@ -0,0 +1,88 @@
+$active-color: #30b6dd;
```

Comment:
you use this color also in header component, maybe it is better to remove it to common style project ?